src/Components/v2/CarouselV3.tsx

```diff
@@ -11,10 +11,15 @@ interface CarouselProps {
    */
   data: any
 
+  /**
+   * If this carousel contains only one image on render set to true (for SSR)
+   */
+  isOneSlide?: boolean
```

Comment:
This is an implementation detail rather than a consumer API prop. Stuff related to SSR should be entirely hidden, rather than creating a scaffolding in order to work around a problem.